### ISSUE TITLE
search page tabs don't announce open/closed state

### ISSUE THREAD

**Issue by thatbudakguy** [Princeton-CDH/mep-django#641]
from [john's review](https://docs.google.com/spreadsheets/d/1N-oBy9Mp91TEdXEi6rjCMNsbJNacxMh6CEGTHpCRfws/edit?usp=sharing):
> The ""Demographics"" tab drop does not announce its state; likely triggered by the following errors: 

> "An element with role=tab must be contained in, or owned by, an element with role=tablist"
"Attribute :aria-setsize not allowed on element."

> Implement remaining tablist aria attributes as per 
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Tab_Role.
